src/Http/Http.Abstractions/test/EndpointFilterInvocationContextOfTTests.cs

```diff
@@ -64,6 +56,16 @@ public void AllowsEnumerationOfParameters()
         Assert.Equal(4, enumeratedCount);
     }
 
+    [Fact]
+    public void HandlesIListReadOperations()
+    {
+        var context = new EndpointFilterInvocationContext<string, int, bool>(new DefaultHttpContext(), "This is a test", 42, false);
```

Comment:
I guess adding a nullable type with a value to the constructor here would validate my comment.